clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupIngressConverter.groovy

```diff
@@ -130,4 +127,48 @@ class SecurityGroupIngressConverter {
       }
     }.flatten().unique()
   }
+
+  static Map<String, List> computeLatestIpRules(List<IpPermission> newList, List<IpPermission> existingRules) {
+    List<IpPermission> tobeAdded = new ArrayList<>()
+    List<IpPermission> tobeRemoved = new ArrayList<>()
+    List<IpPermission> tobeUpdated = new ArrayList<>()
+    List<IpPermission> filteredList = newList.findAll { elements -> elements.userIdGroupPairs.size() == 0 }
```

Comment:
nit: I prefer to either use something completely non-descriptive like the groovy `it` or (preferably) something descriptive like `ipPermission` 
(and same below)
(also since you have this nice convention already... `filteredNewList`)
```suggestion
    List<IpPermission> filteredNewList = newList.findAll { ipPermission -> ipPermission.userIdGroupPairs.size() == 0 }
```